### PR TITLE
examples: add a ListView: applications launcher

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: examples/flow_box/main.rs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+use gtk::prelude::*;
+
+fn create_color_button(color: &'static str) -> gtk::Button {
+    let button = gtk::Button::new();
+    let drawing_area = gtk::DrawingAreaBuilder::new()
+        .content_height(24)
+        .content_width(24)
+        .build();
+    drawing_area.set_draw_func(move |_, cr, _width, _height| {
+        let rgba = gdk::RGBA::from_str(color).unwrap();
```
Comment (sdroege) [inline examples/flow_box/main.rs:14]: This should probably happen outside the closure, once. Instead of inside the closure for each frame that is drawn :)

**Review by sdroege — COMMENTED** [review]
Looks good to me otherwise

File: examples/flow_box/main.rs (hunk 1 above)
Comment (bilelmoussaoui) [reply examples/flow_box/main.rs:14]: Oh indeed.

**Review by sdroege — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
I will go ahead and merge it despite the docs job failing, till rustup gets back to normal

**Pull request merged**